cmd/initialize.go

```diff
@@ -25,9 +29,34 @@ var initializeCmd = &cobra.Command{
 
 func initializeDatabase() {
 	runDatabaseEffects(func(db *gorm.DB) {
-		if db.Migrator().CreateTable(database.Tables...) != nil {
+		if db.AutoMigrate(&table.Workflow{}, &table.ScheduledWorkflow{}, &table.WorkflowSchedulerLock{}) != nil {
```

Comment:
let's have an explicit auto-migrate command in addition to initialize

Reply:
`db.AutoMigrate()` included creating the newly added indices on workflows table's combined column of `name` and `artifact`.  Applying `initialize` first time and subsequent times give correct schema whether before or after adding the unique index `workflows_name_artifact` in the `database/table/tables.go`.